Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]

error: implicitly declaring library function 'isspace' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
--- configure.orig	2008-02-27 22:35:01.000000000 -0600
+++ configure	2020-12-10 02:54:53.000000000 -0600
@@ -3450,6 +3450,7 @@
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
+#include <stdlib.h>
 #include <ctype.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
--- generic/tix.h.orig	2008-02-27 22:35:16.000000000 -0600
+++ generic/tix.h	2020-12-10 03:23:50.000000000 -0600
@@ -34,6 +34,7 @@
 #include <tcl.h>
 #endif
 
+#include <ctype.h>
 #ifdef HAVE_STRING_H
 #include <string.h>
 #endif
